paper-radio-group.html

```diff
@@ -71,116 +69,77 @@
 
     properties: {
       /**
-       * Fired when the selected element changes to user interaction.
-       *
-       * @event paper-radio-group-changed
+       * If you want to use the attribute value of an element for `selected` instead of the index,
+       * set this to the name of the attribute.
        */
-
-      /**
-       * Gets or sets the selected element. Use the `name` attribute of the
-       * <paper-radio-button> that should be selected.
-       *
-       * @attribute selected
-       * @type String
-       * @default null
-       */
-
-      selected: {
+      attrForSelected: {
         type: String,
-        value: null,
-        notify: true,
-        reflectToAttribute: true,
-        observer: "_selectedChanged"
+        value: "name",
+        readOnly: true
       }
     },
 
     keyBindings: {
-      'left up': '_selectPrevious',
-      'right down': '_selectNext',
+      'left up': 'selectPrevious',
+      'right down': 'selectNext',
     },
 
-    _selectedChanged: function() {
-      // TODO: This only needs to be async while a domReady event is unavailable.
+    _updateSelected: function() {
+      Polymer.IronSelectableBehavior._updateSelected.apply(this);
       this.async(function() {
-        this._selectIndex(this._valueToIndex(this.items, this.selected));
+        this._valueToItem(this.selected).checked = true;
```

Comment:
I think you can eliminate this and just set the value of `selectedAttribute` to `checked` to have `IronSelectable` manages the `checked` property for you. e.g.
```
selectedAttribute: {
  value: 'checked'
}
```